src/Microsoft.Extensions.Caching.Memory/CacheEntry.cs

```diff
@@ -52,12 +53,12 @@ internal class CacheEntry : ICacheEntry
             {
                 throw new ArgumentNullException(nameof(notifyCacheOfExpiration));
             }
-            
+
             Key = key;
             _notifyCacheEntryDisposed = notifyCacheEntryDisposed;
             _notifyCacheOfExpiration = notifyCacheOfExpiration;
 
-            CacheEntryHelper.EnterScope(this);
+            _scopes = CacheEntryHelper.EnterScope(this);
```

Comment:
Better. Why not just `_scope`? There's only one of them as far as the CacheEntry is concerned.